lib/platform/azure/index.ts

```diff
@@ -127,7 +128,22 @@ export async function getRawFile(
     repoId = config.repoId;
   }
 
-  const buf = await azureApiGit.getItemContent(repoId, fileName);
+  const versionDescriptor: GitVersionDescriptor = <GitVersionDescriptor>{
+    version: branchOrTag,
+  };
```

Comment:
```suggestion
  const versionDescriptor: GitVersionDescriptor = {
    version: branchOrTag,
  } as GitVersionDescriptor;
```

please don't use deprecated typescript cast operator.